### ISSUE TITLE
WaterHeater source side flow rate sizing requires plant supply to be >58C with no explanation

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#4792]
Helpdesk ticket 10340
** Severe ** Autosizing of Source side water heater design flow rate requires Sizing:Plant object to have an exit temperature >= 58C
** ~~~ ** Occurs for WaterHeater:Mixed object=TANK ACS
** Fatal ** Preceding sizing errors cause program termination

How can I solve this problem, considering that the sizing temperature for the HW loop should be 50°C and not 58° or more?

Workarounds are to change the HW loop sizing to 58C to satisfy this hard-wired limit in the source code, or hard-size the water heater source side flow rate.

**Comment by mjwitte** [comment]
Why is this limit there?
Can this be a warning instead of a fatal error?
Or can it be removed or be made an input in waterheater:sizing?
If it remains, the I/O Ref and Engineering Ref should say what it is and why it's there.
Same goes the the <8C limit when sizing the source flow rate for a chilled water tank.

**Comment by EnergyArchmage** [comment]
yes the input could be moved into the water heater sizing object. The sizing input was simplified by assuming start and stop temperatures from ASHRAE handbook (simplifying input for sizing is an overridding theme).  If the finish temperature can never be reached it is a fatal problem. But the main plant sizing exit temperature doesn't need to do double duty here, what is another input field.  Four more input fields to not hard code the start and stop tank temperatures.   

		if ( ! WaterThermalTank( WaterThermalTankNum ).IsChilledWaterTank ) {
			Tstart = 14.44;
			Tfinish = 57.22;
		} else {
			Tstart = 14.44;
			Tfinish = 9.0;
		}

**Comment by mjwitte** [comment]
Also Helpdesk Ticket 10460
There are similar limits in place for chilled water storage tanks.  And these can be tripped over even if the user specifies a hard-size because we now try to calculate the autosized value and report it along with the user-specified value.  At a minimum, perhaps these should be changed to warnings that simply set the autosized value to zero.  Added the user file from this ticket.
